### PR TITLE
:sparkles: Add location based stats

### PR DESCRIPTION
**Pull request by Ked57**
Code might not be the best but at least it works

### CODE REVIEW

File: ui/src/index.html [hunk 1]
```diff
@@ -1,139 +1,191 @@
 <!DOCTYPE html>
 <html>
```
Comment (Gillespie59) [inline ui/src/index.html]: Could you add a lang="en" attribute on this tag for accessibility purpose

File: ui/src/index.html [hunk 2]
```diff
@@ -1,139 +1,191 @@
...
-        <button class="button button--no-that-great" type="button" id="submitNotThatGreat">
-          <span class="button__emoji">😐</span>
-          Not that great
-        </button>
-        <button class="button button--not-great-at-all" type="button" id="submitNotGreatAtAll">
-          <span class="button__emoji">😤</span>
-          Not great at all
-        </button>
+      <div id="homePage" class="page hidden">
+        <h2>Welcome, <span id="userId"></span></h2>
```
Comment (Gillespie59) [inline ui/src/index.html]: a h2 before h1 ?
Could your remove the h2, and use a <p> tag with the right CSS styles ?

File: ui/src/index.html [hunk 3]
```diff
@@ -1,139 +1,191 @@
...
-    </div>
-    <div id="recordingPage" class="page hidden">
-      <h2>Sit tight!</h2>
-      <h1>I'm recording your vote...</h1>
-      <div class="loader">
-        <svg class="circular" viewBox="25 25 50 50">
-          <circle class="path" cx="50" cy="50" r="20" fill="none" strokeWidth="2" strokeMiterlimit="10" />
-        </svg>
+      <div id="thankYouPage" class="page hidden">
+        <h2>Thanks!</h2>
```
Comment (Gillespie59) [inline ui/src/index.html]: a h2 before h1 ?
Could your remove the h2, and use a

tag with the right CSS styles ?

File: ui/src/index.html [hunk 4]
```diff
@@ -1,139 +1,191 @@
...
-      <h2>Thanks!</h2>
-      <h1>Your answer was properly recorded.</h1>
-      <div class="button_wrapper">
-        <button class="button button--display-stats" type="button" id="homeButton2">
-          <span class="button__emoji">
-            🏠
-          </span> Home
-        </button>
+      <div id="noCampaignPage" class="page hidden">
+        <h2>We're closed!</h2>
```
Comment (Gillespie59) [inline ui/src/index.html]: idem

File: ui/src/index.html [hunk 5]
```diff
@@ -1,139 +1,191 @@
...
-      <h2>We're closed!</h2>
-      <h1>Looks like you've already voted this month.</h1>
-      <div class="button_wrapper">
-        <button class="button button--display-stats" type="button" id="homeButton3">
-          <span class="button__emoji">
-            🏠
-          </span> Home
-        </button>
+      <div id="alreadyVotedPage" class="page hidden">
+        <h2>We're closed!</h2>
```
Comment (Gillespie59) [inline ui/src/index.html]: idem

File: ui/src/index.html [hunk 6]
```diff
@@ -1,139 +1,191 @@
...
-      <p>If this keeps happening, contact dreamlab@zenika.com or #dreamlab on Slack.</p>
-      <p>You may check the console for more details about the error.</p>
-      <div class="button_wrapper">
-        <button class="button button--display-stats" type="button" id="homeButton4">
-          <span class="button__emoji">
-            🏠
-          </span> Home
-        </button>
+      <div id="errorPage" class="page hidden">
+        <h2>There was an error!</h2>
```
Comment (Gillespie59) [inline ui/src/index.html]: idem

File: ui/src/index.html [hunk 7]
```diff
@@ -1,139 +1,191 @@
...
-      <p>Come back tommorow.</p>
-      <p>If this keeps happening, contact dreamlab@zenika.com or #dreamlab on Slack.</p>
-      <div class="button_wrapper">
-        <button class="button button--display-stats" type="button" id="homeButton5">
-          <span class="button__emoji">
-            🏠
-          </span> Home
-        </button>
+      <div id="unknownEmployeePage" class="page hidden">
+        <h2>Hi!</h2>
```
Comment (Gillespie59) [inline ui/src/index.html]: idem

File: ui/src/index.html (hunk 1 above)
Comment (Ked57) [reply ui/src/index.html]: done

File: ui/src/index.html (hunk 2 above)
Comment (Ked57) [reply ui/src/index.html]: I agree this is weird, i'm gonna swap those

**Pull request merged**